testutil/src/main/resources/log4j2-test.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Configuration status="WARN">
   <Properties>
-    <Property name="root.log.level">INFO</Property>
+    <Property name="root.log.level">DEBUG</Property>
```

Comment:
You sure about this one?

Reply:
no, but could be useful to enable more logging for the unit tests in another PR